### ISSUE TITLE
Add ability to include additional properties for the order-in-* config

### ISSUE THREAD

**Issue by Gowiem** [ember-cli/eslint-plugin-ember#158]
Just getting around to adding this plugin to my eslint config and I definitely dig it and agree with a lot of the defaults. One thing that is bugging me though is the `order-in-*` config doesn't seem to allow much leeway for project specific properties. My main example and issue here is I use [`ember-cli-prop-types`](https://github.com/crystal-ball/ember-cli-prop-types) to provide a development runtime contract for my components. I place the `propTypes` key at the bottom of each Component like so: 
```js
import Ember from 'ember';
import PropTypes from 'prop-types';
const { number, string, func, bool } = PropTypes;

export default Ember.Component.extend({
  // Code code code...

  propTypes: {
    idx: number.isRequired,
    value: string,
    flushValue: func.isRequired,
    disabled: bool,
  }
});
```

I'd like to keep these here and it'd be awesome to do without having to include an eslint skip for each of these declarations. I originally tried (without reading the code) to just include `'propTypes'` in my own ordering in my eslint config, but that didn't work. I'd like to be able to do something like that. For example this is what I tried and didn't have it work: 

```js
    'ember/order-in-components': [2, {
      order: [
        'service',
        'property',
        'single-line-function',
        'multi-line-function',
        'observer',
        'init',
        'didReceiveAttrs',
        'willRender',
        'didInsertElement',
        'didRender',
        'didUpdateAttrs',
        'willUpdate',
        'didUpdate',
        'willDestroyElement',
        'willClearRender',
        'didDestroyElement',
        'actions',
        ['method', 'empty-method'],
        'propTypes'
      ]
    }]
```

I would love if the above would work as I had expected. Or we could even expect a special identifier to support the above so something like `custom:$PROPERTY_NAME` where in my case the property name would be `propTypes`. 

Would be happy to take a crack at this if you folks think this is something you'd like to support. For now I'll go with include an eslint skip for each of my propType declarations. 🤕 😭

**Comment by rwjblue** [comment]
Seems reasonable to be able to completely control the ordering so that you can insert customizations before/after existing ordered properties.

👍

**Comment by bmish** [comment]
Should be fixed in #665.
